internal/codegen/golang/opts/options.go

```diff
@@ -39,9 +39,10 @@ type Options struct {
 	OmitUnusedStructs           bool     `json:"omit_unused_structs,omitempty"`
 	BuildTags                   string   `json:"build_tags,omitempty"`
 
-	// Unused but included in marshaled json we receive
-	Overrides json.RawMessage `json:"overrides,omitempty"`
-	Rename    json.RawMessage `json:"rename,omitempty"`
+	QuerySetOverrides []Override      `json:"overrides,omitempty"`
+	QuerySetRename    json.RawMessage `json:"rename,omitempty"` // Unused, TODO merge with req.Settings.Rename
+
+	Overrides []GoOverride `json:"-"`
```

Comment:
We consolidate all overrides here and only use overrides parsed from JSON options and overrides coming in from `req.Settings.Overrides` within `ParseOpts()` below to build this slice.